packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -2733,10 +2733,7 @@ describe('Profiler', () => {
         if (__DEV__) {
           expect(console.error).toHaveBeenCalledTimes(1);
           expect(console.error.calls.argsFor(0)[0]).toContain(
-            'Warning: %s\n\nThe fix is to split the update',
-          );
-          expect(console.error.calls.argsFor(0)[1]).toContain(
-            'A user-blocking update was suspended by:',
+            'Warning: A user-blocking update was suspended by:',
           );
           expect(console.error.calls.argsFor(0)[1]).toContain('AsyncText');
```

Comment:
I know this code already existed but curious — any reason we’re not using `toWarnDev()` for this warning?

Reply:
I think it was easier at the time to use console.error, because there was a lot of them. Do you prefer `toWarnDev()`?